### PR TITLE
a few minor changes to the typography design

### PR DESCRIPTION
**Pull request by MelSumner**
A few minor changes to the heading styles based on team & designer feedback.

### CODE REVIEW

File: source/stylesheets/blog.css.scss [hunk 1]
```diff
@@ -132,12 +132,16 @@ article.blog-post {
     font-size: 1em;
     font-weight: normal;
     text-transform: none;
+
```
Comment (locks) [inline source/stylesheets/blog.css.scss]: extraneous newline? 💅

File: source/stylesheets/blog.css.scss [hunk 2]
```diff
@@ -132,12 +132,16 @@ article.blog-post {
...
   }
 
   .blog-post-author {
     color: $black;
   }
-
+  .blog-post-date {
```
Comment (locks) [inline source/stylesheets/blog.css.scss]: do you want these close together or can we add a newline before? 💅

File: source/stylesheets/pages/blog.css.scss [hunk 3]
```diff
@@ -142,3 +143,7 @@
     }
   }
 }
+.api-info-box {
```
Comment (locks) [inline source/stylesheets/pages/blog.css.scss]: missing newline before? 💅

File: source/stylesheets/pages/blog.css.scss [hunk 4]
```diff
@@ -142,3 +143,7 @@
...
+  background-color: #eeeeee;
+  padding: 30px 50px; 
+}
```
Comment (locks) [inline source/stylesheets/pages/blog.css.scss]: how are we doing for the other files? trailing newline or no?

**Comment by MelSumner** [comment]
Okay I fixed the nitpicks I think. 

Is the design more to your liking now?

**Comment by locks** [comment]
Looking sharper, thank you!

**Pull request merged**